Formula/vile.rb

```diff
@@ -0,0 +1,27 @@
+class Vile < Formula
+  desc "Vi Like Emacs Editor"
+  homepage "https://invisible-island.net/vile/"
+  url "https://invisible-island.net/archives/vile/current/vile-9.8v.tgz"
+  sha256 "240edec7bbf3d9df48b3042754bf9854d9a233d371d50bba236ec0edd708eed5"
+  license "GPL-2.0-or-later"
+
+  depends_on "groff" => :build
+  uses_from_macos "flex" => :build
+
+  uses_from_macos "libiconv"
+  uses_from_macos "ncurses"
+
+  def install
+    system "./configure", "--prefix=#{prefix}",
```

Comment:
```suggestion
    system "./configure", *std_configure_args,
```

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/pull/106086/commits/0b036532ece5374dfa7d0b915f54b87c03f828b4.